static/js/app.js

```diff
@@ -5,21 +5,23 @@ var testContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis
 
 var textArea = document.querySelector('#doc_view');
 var currentDocument = new Doc("test", testContent);
-var currentHighlighterColor;
-// future implementation
+var currentHighlighterColor = $('.color-selected')[0].id;
```

Comment:
Referencing a variable by array index should generally be a last resort. Makes code hard to read and isn't descriptive.

Reply:
removed reference by array index